### PR TITLE
Fix: Updated Broadcast receiver to do nothing if key is null

### PR DESCRIPTION
**Pull request by aanorbel**

### CODE REVIEW

File: app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java [hunk 1]
```diff
@@ -49,6 +49,10 @@ public TestRunBroadRequestReceiver(PreferenceManager preferenceManager, EventLis
     public void onReceive(Context context, Intent intent) {
         String key = intent.getStringExtra("key");
         String value = intent.getStringExtra("value");
+        // If either key is null, do nothing
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java:52]: Here you say "either": which other key should we be checking below?

Comment (aanorbel) [reply app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java:52]: None.

Comment (aanorbel) [inline app/src/main/java/org/openobservatory/ooniprobe/receiver/TestRunBroadRequestReceiver.java:52]:
```suggestion
        // If key is null, do nothing
```

File: app/src/main/java/org/openobservatory/ooniprobe/common/service/RunTestService.java [hunk 2]
```diff
@@ -209,6 +209,9 @@ public void onReceive(Context context, Intent intent) {
             ) {
                 return;
             }
+            if (key == null) {
```
Comment (aanorbel) [inline app/src/main/java/org/openobservatory/ooniprobe/common/service/RunTestService.java:212]:
```suggestion
            // If key is null, do nothing
            if (key == null) {
```

**Review by bassosimone — APPROVED** [review]
🐳

**Pull request merged**